docs/Course Wiki/ECE Course Offerings/ECE120.md

```diff
@@ -0,0 +1,55 @@
+# ECE120
+
+## Content Covered
+
+- Representation of information
+- Boolean algebra
+- Combinational network analysis and design
+- Sequential network analysis and design
+- Finite state machine and memory design
+- Computer organization and machine-level programming
+- Programming in C
+
+The course starts by covering the many different representations of information in the form of bits in a computing system, then moves on to cover Boolean algebra as it relates to digital networks, combinational and sequential circuits, finite state machines and memories, and ends with a basic overview of computer organization and assembly programming principles using a basic educational ISA known as LC-3. Some C programming is also covered in the first few weeks through a series of small labs, but no prior programming experience is needed and mastery of C is not expected by the end of this course, as [ECE220](ECE220.md) (the next course in line after ECE120) starts from where this course leaves off and covers C in much more detail. 
+
+## Prerequisites
+
+There are no technical prerequisites to take ECE120 as it is an introductory course to computing fundamentals, and aims to provide an overview of all the content encapsulated by the field, which future classes will expand on by diving deeper into specific topics, such as low level programming (ECE220), [ECE391](ECE391.md)), digital design ([ECE385](ECE385.md)), and computer organization ([ECE411](ECE411.md)).
```

Comment:
this sentence is pretty long, can we split it

Reply:
Split the sentence into two, should be resolved on newest commit.